integration_test/gov_module/gov_proposal_test.yaml

```diff
@@ -19,8 +19,8 @@
     - cmd: seid q gov proposal $PROPOSAL_ID --output json | jq -r .status
     - cmd: printf "12345678\n" | seid tx gov vote $PROPOSAL_ID yes --from node_admin --chain-id sei --fees 2000usei -b block -y --output json | jq -r .code
       node: sei-node-1
-    # since quorum is 0.5, we only need 2/4 votes and expect proposal to pass after 35 seconds
-    - cmd: sleep 35
+    # since quorum is 0.5, we only need 2/4 votes and expect proposal to pass after 105 seconds
```

Comment:
why do we bump this time?

Reply:
@yzang2019 I increased the deposit time for gov proposals here since it makes it easier to run the EVM precompile integration test for gov deposit.

Reply:
@yzang2019 I increased the deposit time  for gov proposals here since it makes it easier to run the EVM precompile integration test for gov deposit